2-mandatory/3-financial-times.js

```diff
@@ -14,7 +20,13 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  let str = allArticleTitles[0];
```

Comment:
Check the function comment again - the editor wants titles with the fewest _words_, not the fewest _characters_.

Reply:
Yes I should have done that